### PR TITLE
release: create v0.16.2 branch

### PR DESCRIPTION
**Pull request by Roasbeef**
Includes the following hot fix PRs: 
  * https://github.com/lightningnetwork/lnd/pull/7641
  * https://github.com/lightningnetwork/lnd/pull/7637
  * https://github.com/lightningnetwork/lnd/pull/7647

### CODE REVIEW

**Review by yyforyongyu — APPROVED** [review]

**Review by guggero — APPROVED** [review]
LGTM :tada:

File: build/version.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ const (
 	AppMinor uint = 16
 
 	// AppPatch defines the application patch for this binary.
-	AppPatch uint = 1
+	AppPatch uint = 2
```
Comment (Crypt-iQ) [inline build/version.go:46]: Does this need to be updated anywhere else?

**Review by Crypt-iQ — APPROVED** [review]
LGTM

File: build/version.go (hunk 1 above)
Comment (Roasbeef) [reply build/version.go:46]: Nope, everything else (cli, etc) will read this value.

**Review by Crypt-iQ — APPROVED** [review]

**Pull request merged**